docs/libcurl/curl_share_setopt.3

```diff
@@ -43,6 +43,11 @@ sure that the callback uses a different lock for each kind of data.
 \fIaccess\fP defines what access type libcurl wants, shared or single.
 
 \fIuserptr\fP is the pointer you set with \fICURLSHOPT_USERDATA\fP.
+
+.IP CURL_LOCK_DATA_KNOWNHOSTS
+Share the known_hosts handle for SFTP and SCP transfers. 
```

Comment:
Maybe elaborate slightly more on what "the known_hosts handle" is for users who don't know libcurl internals?